.github/workflows/codeql-analysis.yml

```diff
@@ -67,12 +150,16 @@ jobs:
         with:
           name: results-${{ matrix.language }}
           path: results/*.sarif
+          overwrite: true
       - name: Warn if analysis fails
         if: failure()
         uses: peter-evans/create-or-update-comment@v4
         with:
           issue-number: ${{ github.event.issue.number }}
           body: |
-            Security analysis has failed for this add-on.
-            [See GitHub workflow](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})
-            Please, contact NV Access for more details.
+            Security analysis has reported warnings for this add-on.
+            This won't prevent its  acceptance on the store, but you may want to review warnings to improve the quality of your code for future versions.
+            You can open this link and [download artifacts containing the results of the analysis](${{ steps.uploadResults.outputs.artifact-url }}). Unzip artifacts and open the .sarif file in your preferred editor. For example, you can use [Microsoft's Sarif web based reader](https://microsoft.github.io/sarif-web-component).
+            Please, don't close this issue.
+            Your add-on will be accepted soon.
```

Comment:
is this necessary? shouldn't this issue be closed as the actions continue and the PR accepting the add-on has merged

Reply:
OK, I'll remove this comment